code/presentation/mb-client/package.json

```diff
@@ -58,12 +59,14 @@
     "expo-dev-client": "~2.4.11",
     "expo-image": "~1.3.4",
     "expo-localization": "~14.3.0",
+    "expo-location": "^16.5.3",
     "expo-splash-screen": "0.20.5",
     "expo-status-bar": "~1.6.0",
     "expo-system-ui": "~2.4.0",
     "i18next": "^22.4.11",
     "lodash.memoize": "^4.1.2",
     "nativewind": "^2.0.10",
+    "pnm": "^0.0.0",
```

Comment:
what is this library? are we even using it?

Reply:
Removed by using terminal